pkg/factory/risingwave_object_factory.go

```diff
@@ -177,6 +179,8 @@ func (f *RisingWaveObjectFactory) componentName(component, group string) string
 		return f.risingwave.Name + "-compactor" + groupSuffix(group)
 	case consts.ComponentConnector:
 		return f.risingwave.Name + "-connector" + groupSuffix(group)
+	case consts.ComponentStandalone:
+		return f.risingwave.Name
```

Comment:
Should we also suffix it?
```suggestion
		return f.risingwave.Name + "-standalone"
```

Reply:
Good point!